WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -355,6 +355,8 @@ + (NSArray *)routes
       FB_SETTING_USE_CLEAR_TEXT_SHORTCUT: @([FBConfiguration useClearTextShortcut]),
       FB_SETTING_INCLUDE_HITTABLE_IN_PAGE_SOURCE: @([FBConfiguration includeHittableInPageSource]),
       FB_SETTING_INCLUDE_NATIVE_FRAME_IN_PAGE_SOURCE: @([FBConfiguration includeNativeFrameInPageSource]),
+      FB_SETTING_INCLUDE_PLACEHOLDER_VALUE_IN_PAGE_SOURCE: @([FBConfiguration includePlaceholderValueInPageSource]),
```

Comment:
this would be a breaking change since we already have this attribute included into the page source. 

It is ok to only have a setting for minmax attributes because:
- It's a part of this PR
- placeholderValue is a native snapshot attribute and thus is retrieved together with element snapshot while minmax, visibility, etc must perform additional accessbility call to fetch the value.